plugins/abnormal_security/help.md

```diff
@@ -63,104 +139,106 @@ Example output:
-                "Suspicious Link",
-                "Unusual Sender Domain"
-            ],
-            "remediationTimestamp": "2020-05-17T21:09:55Z",
-            "isRead": false,
-            "attackedParty": "Employee (other)",
-            "abxMessageId": -1111111111111111112,
-            "abxPortalUrl": "https://portal.abnormalsecurity.com/home/threat-center/remediation-history/-1111111111111111112",
-            "subject": "[LIKELY PHISHING] Testing Myths",
-            "fromAddress": "user@example.com",
-            "fromName": "Test Example",
-            "toAddresses": "user@example.com",
-            "recipientAddress": "user@example.com",
-            "receivedTime": "2020-05-17T21:09:51Z",
-            "sentTime": "2020-05-17T21:09:46Z",
-            "internetMessageId": "<user@example.com>",
-            "urls": [
-                "https://example.com",
-                "https://example.com"
-            ]
-        },
-        {
-            "threatId": "184712ab-6d8b-47b3-89d3-a314efef79e2",
-            "autoRemediated": false,
-            "postRemediated": false,
-            "attackType": "Spam",
-            "attackStrategy": "Name Impersonation",
-            "returnPath": "user@example.com",
-            "senderIpAddress": "192.168.50.1",
-            "impersonatedParty": "none / Others",
-            "attackVector": "Link",
-            "summaryInsights": [
-                "Abnormal Email Body HTML",
-                "Suspicious Link",
-                "Unusual Sender Domain",
-                "Unusual Sender"
-            ],
-            "attackedParty": "Employee (other)",
-            "abxMessageId": -1111111111111111114,
-            "abxPortalUrl": "https://portal.abnormalsecurity.com/home/threat-center/remediation-history/-1111111111111111114",
-            "subject": "The Truth About 7 Myths",
-            "fromAddress": "user@example.com",
-            "fromName": "Test Example",
-            "toAddresses": "user@example.com",
-            "recipientAddress": "user@example.com",
-            "receivedTime": "2020-05-10T21:05:08Z",
-            "sentTime": "2020-05-10T21:05:03Z",
-            "internetMessageId": "<user@example.com>"
-        }
+      {
+        "threatId": "184712ab-6d8b-47b3-89d3-a314efef79e2",
+        "autoRemediated": true,
+        "postRemediated": false,
+        "attackType": "Spam",
+        "attackStrategy": "Unknown Sender",
+        "returnPath": "user@example.com",
+        "senderIpAddress": "192.168.50.1",
+        "impersonatedParty": "none / Others",
+        "attackVector": "Link",
+        "summaryInsights": [
+          "Abnormal Email Body HTML",
+          "Suspicious Link",
+          "Unusual Sender Domain",
+          "Abnormal Email SignOff",
+          "Unusual Sender"
+        ],
+        "remediationTimestamp": "2021-02-02T21:09:41Z",
+        "isRead": false,
+        "attackedParty": "Other",
+        "abxMessageId": 1111111111111111113,
```

Comment:
We may want to make these strings to avoid JS frontend rounding bug

Reply:
Also lines 197 and 229 (is it supposed to be a negative value? Or is it an instance of an overflow?).
Please also make sure the plugin output handles converting this ID into a string.

Reply:
Hey @mberezin-r7,

For this one only the formatting has been updated in help.md by `make regenerate` command - this output belongs to another action (not part of this PR), I'm happy to make a code change, but it would require code update and testing for Get Threat Details action. If this is still something we want to do, could we create/get a ticket created for that change?

Thanks for the review! :)

Reply:
Sorry for the delay in my response! This comment request is not a major enough change to suggest an additional ticket. I acknowledge that this request is out of scope and accept the code as is.